tremor-connectors/src/impls/http.rs

```diff
@@ -205,4 +205,4 @@ pub mod meta;
 /// HTTP server connect
 pub mod server;
 /// HTTP utils
-pub(crate) mod utils;
+pub mod utils;
```

Comment:
Let's keep this crate private, there is virtually no re-use and it pollutes the public interface which now that we made crates public is a concearn

Reply:
Done